cmd/proxy/actions/app.go

```diff
@@ -6,9 +6,9 @@ import (
 
 	"github.com/gobuffalo/buffalo"
 	"github.com/gobuffalo/buffalo/render"
-	forcessl "github.com/gobuffalo/mw-forcessl"
-	paramlogger "github.com/gobuffalo/mw-paramlogger"
-	sessions "github.com/gobuffalo/x/sessions"
+	"github.com/gobuffalo/mw-forcessl"
```

Comment:
These three imports should not be changed, unless their respective uses are also updated.

Reply:
I double checked that and I see that they are redundant. `github.com/gobuffalo/mw-forcessl` is `package forcessl` and `github.com/gobuffalo/mw-paramlogger` is `package paramlogger`. I can revert, but with this changes everything still works :slightly_smiling_face:

Reply:
Oh weird, just the import path doesn't match. I was wondering why the build didn't fail.